config/config.go

```diff
@@ -157,6 +158,8 @@ type TracingConfig struct {
 	Enabled   bool   `yaml:"enabled"`
 	Namespace string `yaml:"namespace"`
 	Service   string `yaml:"service"`
+	Port      int32  `yaml:"port"`
+	Protocol  string `yaml:"protocol"`
```

Comment:
Hmm Protocol is not used or are you going to add it later?

Reply:
Removed. Not adding it now.